tests/publiccloud/run_ltp.pm

```diff
@@ -19,6 +19,8 @@ use utils;
 use repo_tools 'generate_version';
 use Mojo::UserAgent;
 
+our $root_dir = '/root';
```

Comment:
why you think it is better than `~/` ? I am not 100% sure but since it is root most probably we already at `/root` folder

Reply:
If I'm not wrong `~/` gets evaluated on globbing. But I had some problems with it. So a absolute path makes world more strict/easy for me.

Reply:
in ec2 we are not root by default.

Reply:
but it is our working dir on the helper VM. And there we are root.

If you have problems with this. We could move to 
```
mkdir -p /tmp/

Reply:
I am fine with /root/ location